secboot/secboot_tpm.go

```diff
@@ -69,3 +76,80 @@ func checkSecureBootEnabled() error {
 
 	return nil
 }
+
+const tpmPCR = 12
+
+type TPM struct {
```

Comment:
This should be renamed to something technology-agnostic if used to build an abstraction for different secure boot/key storage methods.